app/views/layouts/application.html.slim

```diff
@@ -1,5 +1,5 @@
 doctype html
-html lang=I18n.locale data-default-locale=I18n.default_locale
+html lang=I18n.locale data-default-locale=I18n.default_locale data-events-enabled=CodeOcean::Config.new(:code_ocean).read.dig('codeocean_events', 'enabled') data-flowr-enabled=CodeOcean::Config.new(:code_ocean).read.dig(:flowr, :enabled) data-flowr-answers-per-query=CodeOcean::Config.new(:code_ocean).read[:flowranswers_per_query] || 3
```

Comment:
Can we find a smaller scope for these data attributes? I think they are not needed globally (but only for some routes).

Also, we might need to consider some better caching: With the previous solution, the value was parsed at compile time. Now, it is embedded for each page rendering.

Reply:
> How now I don't think we need any caching strategy.

I see that differently, since each request rendering the application layout would read and parse the same config at least three times.

> The configuration causes me significant trouble. I think I will rewrite it to be in memory and easy to test soon.

Can you specify the trouble, please? Testing is relatively simple using the config and there are mulitple examples where we overwrite specific config values for tests already. This alone shouldn't be the motivation to rewrite the config store. I would think there are other, more important topics right now...

Reply:
Some simple memorization could do the trick, or at least ensuring you are not initializing the config multiple times (There is some memorization built-in already). // cc @nenock

Reply:
Thanks!

Now, there is only an issue with the config key for `flowranswers_per_query`; that doesn't exist, does it?

Reply:
Thanks! I just noticed another discrepancy: For `'data-events-enabled'`, you use strings, while you use symbols for `'data-flowr-enabled'` -- why?

Reply:
I am sorry. I forgot to adjust it when I copied it from the original version.